README.md

```diff
@@ -23,12 +23,7 @@ import Ember from 'ember';
 import StateMixin from 'ember-state-services/mixin'
 
 export default Ember.Object.extend(StateMixin, {
-  stateName: 'emailEdit',
-  setupState: function(Factory, model) {
```

Comment:
I agree. This example isn't really demonstrating anything useful (and potentially adding confusion to a newcomer). We might want to add another section with a better example with how one might use setupState but that is probably more of an advanced use case.